ctk/common/src/main/kotlin/org/codice/compliance/verification/core/CoreVerifier.kt

```diff
@@ -77,33 +71,6 @@ abstract class CoreVerifier(protected val node: Node) {
             }
         }
 
-        /**
-         * Verifies Name Identifiers formatted as type Entity, according to the Core Spec document.
-         *
-         * 8.3.6 Entity Identifier
-         */
```

Comment:
Just moved these two methods to `SamlDefinedIdentifiersVerifier` (which verifies section 8 SAML-Defined Identifiers) because it's under section 8.3.6 Entity Identifier